easybuild/easyconfigs/a/abtem/abtem-1.0.0b24-foss-2020b.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonPackage'
+
+name = 'abtem'
```

Comment:
@schiotz Looks like the official software name is `abTEM`?

Reply:
Fixed.